webapp/src/routes/BPJson.js

```diff
@@ -21,38 +21,16 @@ const eosApi = EosApi({
 
 const getBpJSONOffChain = async (producer) => {
   try {
-    const { data: bpJson } = await axios.get(
-      producer?.owner === 'okcapitalbp1'
-        ? `${producer?.url}/bp.json`
-        : `https://cors-anywhere.herokuapp.com/${producer?.url}/bp.json`,
-      {
-        timeout: 5000
-      }
-    )
+    const { data: bpJson } = await axios.get(`${producer?.url}/bp.json`, {
```

Comment:
the file may be called something different based on the contents of `[chains.json](https://eoscostarica.io/chains.json)` 


for example:

```
{
  "chains": {
    "aca376f206b8fc25a6ed44dbdc66547c36c6c33e3a119ffbeaef943642f0e906": "/bp.json",
    "2a02a0053e5a8cf73a56ba0fda11e4d92e0238a4a2aa74fccf46d5a910746840": "/jungle3.json",
    "384da888112027f0321850a169f737c33e53b388aad48b5adace4bab97f437e0": "/proton.json",
    "71ee83bcf52142d61019d95f9cc5427ba6a0d7ff8accd9e2088ae2abeaf3d3dd": "/proton-testnet.json",
    "4667b205c6838ef70ff7988f6e8257e8be0e1284a2f59699054a018f743b1d11": "/telos.json",
    "1eaa0824707c8c16bd25145493bf062aecddfeb56c736f6ba6397f3195f33c9f": "/telos-testnet.json",
    "1064487b3cd1a897ce03ae5b6a865651747e2e152090f99c1d19d44e01aea5a4": "/wax.json",
    "f16b1833c747c43682f4386fca9cbb327929334a762755ebec17f6f23c9b8a12": "/wax-testnet.json"
  }
}
```

Reply:
if there is a chains json we should use that , if not it could default to just bp.json , we need this to manage our own json files across networks, see example provided